packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -862,6 +862,58 @@ describe('StripeHelper', () => {
       sinon.assert.callCount(mockStatsd.increment, 1);
     });
 
+    it('creates a subscription with automatic tax successfully', async () => {
+      const attachExpected = deepCopy(paymentMethodAttach);
+      const customerExpected = deepCopy(newCustomerPM);
+      sandbox
+        .stub(stripeHelper.stripe.paymentMethods, 'attach')
+        .resolves(attachExpected);
+      sandbox
+        .stub(stripeHelper.stripe.customers, 'update')
+        .resolves(customerExpected);
+      sandbox
+        .stub(stripeHelper.stripe.subscriptions, 'create')
+        .resolves(subscriptionPMIExpanded);
+      const subIdempotencyKey = uuidv4();
+      stripeFirestore.insertCustomerRecordWithBackfill = sandbox
+        .stub()
+        .resolves({});
+      stripeFirestore.insertSubscriptionRecordWithBackfill = sandbox
+        .stub()
+        .resolves({});
+      stripeFirestore.insertPaymentMethodRecord = sandbox.stub().resolves({});
```

Comment:
These fixtures/stubs are identical to the previous test case -- can this be DRYed up a bit, so it's easier to tell that the only difference is the automatic tax?

Reply:
I had considered that, but changing the expected calls and such makes them a bit harder to follow.